tabby-core/src/services/commands.service.ts

```diff
@@ -109,6 +109,6 @@ export class CommandService {
                 description: c.sublabel,
                 icon: c.icon,
             })),
-        )
+        ).catch(() => {return})
```

Comment:
Yeah you're right, I should have place the catch call on a higher side. Same on the other you've point to me.

Corrected with 49f9a10372d15f7e6b358ba9a43536a8e188e8b7 :
In that case, `showSelector` now return the promise return by the selector. I replaced `catch` by `then` because of the type of the promise (`Promise<void>`).